airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination_async/AsyncStreamConsumer.java

```diff
@@ -148,21 +139,6 @@ private void validateRecord(final AirbyteMessage message, final StreamDescriptor
     if (!streamNames.contains(streamDescriptor)) {
       throwUnrecognizedStream(catalog, message);
     }
-
-    trackerIsValidRecord(message, streamDescriptor);
-  }
-
-  private void trackerIsValidRecord(final AirbyteMessage message, final StreamDescriptor streamDescriptor) {
-    // todo (cgardens) - is valid should also move inside the tracker, but don't want to blow up more
-    // constructors right now.
-    try {
-
-      if (!isValidRecord.apply(message.getRecord().getData())) {
```

Comment:
we will have to figure out what to do when we move Redshift over to use this. But for that's the only one that uses it and trying to save this interface that relies on the data being deserialized isn't worth the effort now.